docs/source/onyo.rst

```diff
@@ -0,0 +1,27 @@
+Python API
+============
```

Comment:
Nitpick: length of the `===` does not align with Header :)

Reply:
Yeah, that's due to using `ex` to replace the header in the first line.

But it's a good excuse for me to improve my `ex`-fu. I'll fix this.

Reply:
Fixed and pushed. :-)